step/init.gradle.gotemplate

```diff
@@ -28,8 +28,7 @@ gradle.settingsEvaluated { settings ->
             endpoint = '{{ .Endpoint }}'
             authToken = '{{ .AuthToken }}'
             enabled = true
-            push = true
-            avoidUploads = false
```

Comment:
The `avoidUploads` param has been removed here: https://github.com/bitrise-io/gradle-remote-cache/pull/7